sormas-base/setup/sormas.properties

```diff
@@ -146,6 +146,10 @@ sms.auth.secret=
 #geocodingLongitudeJsonPath=$.features[0].geometry.coordinates[0]
 #geocodingLatitudeJsonPath=$.features[0].geometry.coordinates[1]
 
+# EPSG4326 WKT String used to interpret GEO-Coordinate data (e.g. Axis order)
+# default: GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS 84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0,AUTHORITY[\"EPSG\",\"8901\"]],UNIT[\"degree\",0.01745329251994328,AUTHORITY[\"EPSG\",\"9122\"]],AXIS[\"Long\",EAST],AXIS[\"Lat\",NORTH],AUTHORITY[\"EPSG\",\"4326\"]]
+# geocodingEPSG326_WKTstring=
```

Comment:
missing a `4`, Should be `geocodingEPSG4326_WKTstring` but I also suggest keeping the name for this property simple as `geocodingEPSG4326_WKT`